pkg/report/report.go

```diff
@@ -75,31 +75,18 @@ func (r Report) String() string {
 	return fmt.Sprintf("crash: %v\n%s", r.Title, r.Report)
 }
 
-type Type int
+type Type string
 
 const (
-	Unknown Type = iota
-	Hang
-	MemoryLeak
-	DataRace
-	UnexpectedReboot
+	Unknown          = Type("")
```

Comment:
From Go's perspective, it's better to keep it "" here. Basically that's what was in the old version of code as well -- Unknown was 0.

If you worry about `pkg/report` tests, I can adjust `report_test.go` so that there would be UNKNOWN in `pkg/report/testdata` instead of no `TYPE: ` at all. Though I don't know if we really want to enforce it this way, maybe just require `TYPE: ` to be also present in cases when the `pkg/report` machinery was able to extract a type?

Reply:
> From Go's perspective, it's better to keep it "" here. Basically that's what was in the old version of code as well -- Unknown was 0.

Makes sense.
But then maybe add a String method that will return "UNKNOWN" for `Unknown`, but otherwise just cast the value to string. It will be used with any Printf/Logf.

Reply:
Added a `String()` method.